core/translate/expr.rs

```diff
@@ -1854,7 +1854,7 @@ pub fn translate_expr(
             }
             ast::Literal::String(s) => {
                 program.emit_insn(Insn::String8 {
-                    value: s[1..s.len() - 1].to_string(),
+                    value: s[1..s.len() - 1].replace("''", "'").to_string(),
```

Comment:
Let's move this to a new `sanitize_string()` helper to make it more readable.

Reply:
done